File: translation_output/gpt-3.5/codenet/java/python/p00291.py
```python
def main():
    a = [int(input()) for _ in range(6)]
    b = a[0] + a[1]*5 + a[2]*10 + a[3]*50 + a[4]*100 + a[5]*500
    if b >= 1000:
        print("1")
    else:
        print("0")

if __name__ == "__main__":
    main()
```

File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/python/java/p00291.java
Here is the translated code from Python to Java:

```java
import java.util.Scanner;

public class ChangeCounter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int c1 = scanner.nextInt();
        int c5 = scanner.nextInt();
        int c10 = scanner.nextInt();
        int c50 = scanner.nextInt();
        int c100 = scanner.nextInt();
        int c500 = scanner.nextInt();

        int sum = c1 + c5 * 5 + c10 * 10 + c50 * 50 + c100 * 100 + c500 * 500;

        System.out.println(sum >= 1000 ? 1 : 0);

        scanner.close();
    }
}
```

To test this in a Java environment, you can compile and run this code, then input `3 1 4 0 4 1` when prompted. The expected output would be `0`.
